utilities/transactions/write_prepared_txn_db.cc

```diff
@@ -75,6 +75,22 @@ Status WritePreparedTxnDB::Get(const ReadOptions& options,
                            &callback);
 }
 
+std::vector<Status> WritePreparedTxnDB::MultiGet(
+    const ReadOptions& options,
+    const std::vector<ColumnFamilyHandle*>& column_family,
+    const std::vector<Slice>& keys, std::vector<std::string>* values) {
+  assert(values);
+  size_t num_keys = keys.size();
+  values->resize(num_keys);
+
+  std::vector<Status> stat_list(num_keys);
+  for (size_t i = 0; i < num_keys; ++i) {
+    std::string* value = values ? &(*values)[i] : nullptr;
```

Comment:
I think we can assume `values` is non-null?

Reply:
I agree. Specially that in Get we do assert that it is not null. But in other existing MultiGet implementations it accepts the nullptr and I wanted to be consistent with it.